geo_project_compassion/__manifest__.py

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#       ______ Releasing children from poverty      _
+#      / ____/___  ____ ___  ____  ____ ___________(_)___  ____
+#     / /   / __ \/ __ `__ \/ __ \/ __ `/ ___/ ___/ / __ \/ __ \
+#    / /___/ /_/ / / / / / / /_/ / /_/ (__  |__  ) / /_/ / / / /
+#    \____/\____/_/ /_/ /_/ .___/\__,_/____/____/_/\____/_/ /_/
+#                        /_/
+#                            in Jesus' name
+#
+#    Copyright (C) 2017 Compassion CH (http://www.compassion.ch)
+#    @author: Nathan Flückiger, Emanuel Cino
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,# -*-
+#    coding: utf-8 -*-
+##############################################################################
+
+{'name': 'Geospatial support for compassion projects',
+ 'version': '10.0.1.0.0',
+ 'category': 'GeoBI',
+ 'author': "Fluckiger Nathan",
+ 'depends': [
+     'base',
```

Comment:
You don't need this dependency as `child_compassion` and `base_geoengine` already depend on it.